src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const lis = tree.querySelectorAll('li');
+
+lis.forEach((el) => {
+  const text = el.childNodes[0].nodeValue.trim();
```

Comment:
Consider checking if `el.childNodes[0]` is a text node before accessing `nodeValue`. If it's not a text node, this could lead to unexpected behavior.